wiki/Modding/Forum_modding/id.md

```diff
@@ -0,0 +1,108 @@
+- ![](/wiki/shared/icon/broken-heart.gif) **Broken heart**: beatmap telah didiskualifikasi atau unranked karena beberapa masalah atau tidak mengikuti [ranking criteria](/wiki/Ranking_Criteria).
+- ![](/wiki/shared/icon/nuke.gif) **[Nuke](/wiki/Modding/Nuke)**: beatmap tidak dapat di-rank dengan kondisinya saat itu.
+
+## Komunikasi
+
+### Mod
+
+::: Infobox
+![](img/mod-post.png "Contoh sebuah post mod yang diberikan sebuah kudosu")
+:::
+
+Tidak seperti [diskusi beatmap](/wiki/Beatmap_Discussion), sebuah forum mod post seringkali mencakup seluruh beatmap dan dapat berisi konten yang sangat banyak. Ketika pengguna dapat bebas mengekspresikan pendapatnya dalam bentuk apa saja, secara umum modder mengikuti sebuah struktur spesifik untuk membuat postnya menjadi lebih mudah dibaca:
+
+- Mod dibagi menjadi beberapa bagian yang terdiri dari tingkat kesulitan dan setting umum beatmap.
+- Setiap bagian berisi daftar [timestamp](/wiki/Modding/Timestamp) yang sering menunjuk [hit object](/wiki/Hit_object) secara spesifik.
+- Setiap timestamp diikuti deskripsi singkat masalah pada timestamp tersebut dan terkadang diikuti oleh tangkapan layar untuk memperjelas hal tersebut.
+- Berdasarkan tingkat keparahan masalah, beberapa modder menggunakan [colour coding](/wiki/BBCode#colour) pada postnya untuk menekankan masalah unrankable atau poin penting.
+
+Sistem *review* modern dibuat untuk meniru forum modding pada diskusi beatmap.
+
+### Balasan
+
+::: Infobox
+![](img/mod-response.png "Contoh sebuah balasan dari sebuah mod")
+:::
+
+Meskipun tidak dibutuhkan, mapper diharapkan untuk membalas tiap mod yang masuk. Sebuah balasan biasanya berisi kutipan dari post utama dan diikuti oleh balasannya. Sama seperti mod post, mapper menggunakan dua atau lebih warna font untuk menyampaikan balasannya dan perbedaan warna dapat membantu dalam membedakan masukkan yang diterima dan ditolak.
+
+### Kudosu
+
+*Artikel utama: [Kudosu](/wiki/Modding/Kudosu)*
+
+Jika mod post dianggap membantu, mapper atau anggota QAT/BAT/BN dapat memberikan kudosu pada post tersebut. Berdasarkan peraturan yang tak tertulis, hanya mod post pertama pengguna yang dapat diberikan kudosu. Tanpa memerdulikan berapa banyak masukkan dalam sebuah post atau seberapa berguna bagi mapper, jumlah kudosu yang diterima selalu sama:
+
+- 1 kudosu untuk kondisi umum.
+- 2 kudosu untuk kondisi utas beatmap yang telah tidak aktif lebih dari seminggu. Hal ini bertujuan untuk menggalakkan modding pada map-map tua.
+
+Kudosu berfungsi sebagai ukuran aktivitas seorang modder. Kebanyakan pengguna menggunakan kudosunya untuk meningkatkan [star priority](/wiki/Modding/Star_priority) map yang disukai atau yang ingin dipromosikan.
+
+## Deprecation
+
+Dalam sistem modding berbasis forum, banyak tindakan dilakukan secara manual yang seringkali berujung pada kesalahan. Contoh:
+
+- Umpan balik mapper dapat hilang.
```

Comment:
hmm i don't think the word `lost` here means `hilang` in its literal sense, it's more like `tenggelam di antara sekian banyak mod post yang ada`